test/auth.test.js

```diff
@@ -161,18 +160,17 @@ before(function (cb) {
                 return;
             }
 
-            self.client.ln(self.key, self.link, cb);
```

Comment:
this also removes *snaplink* tests. Did you want those as part of the snaplink change?

Reply:
I guess. I'll do that.